docs/spec/slashing/begin-block.md

```diff
@@ -70,14 +70,17 @@ slashAmount -= slashAmountUnbondings
 slashAmount -= slashAmountRedelegations
 
 curVal.Shares = max(0, curVal.Shares - slashAmount)
+
+signInfo = SigningInfo.Get(val.Address)
+signInfo.JailedUntil = MAX_TIME
+signInfo.Tombstoned = true
```

Comment:
Why do we need to set this?

Reply:
(also if you add a field we need to change the struct definition in the spec)

Reply:
I wanted to get rid of the `SlashedSoFar` tracking.

But to do that, I need to distinguish between whether you're in `Jail` for a liveness fault or byzantine fault.  Because if you're in for a byzantine fault, I should ignore further Evidence.  But if you're in for a liveness fault, I should accept Byzantine evidence.